c7n/resources/directconnect.py

```diff
@@ -20,3 +20,32 @@ class resource_type(TypeInfo):
         permissions_augment = ("directconnect:DescribeTags",)
 
     augment = universal_augment
```

Comment:
I'm not sure if these lines are actually necessary for resources in this file or if they are reflective of older patterns.